habitat-baselines/habitat_baselines/config/social_nav/social_nav.yaml

```diff
@@ -40,8 +40,9 @@ habitat_baselines:
   eval:
     extra_sim_sensors:
       third_rgb_sensor:
-        height: 224
-        width: 171
+        height: 1080
```

Comment:
This is a really large sensor size and will be slow. Are you sure you want the default to be this large?

Reply:
Thanks, done, revert to the original size